docs/developer/getting-started/monorepo-packages.asciidoc

```diff
@@ -41,8 +41,8 @@ yarn kbn watch
 [discrete]
 === List of Already Migrated Packages to Bazel
 
-- @elastic/eslint-config-kibana
-- @elastic/safer-lodash-set
+- @kbn/eslint-config-kibana
+- @kbn/safer-lodash-set
```

Comment:
The rest of this list is sorted alphabetically, so I would suggest that these now gets reordered.